### PR TITLE
DOCS-6311 Changes

### PR DESCRIPTION
**Pull request by rddheath**

### CODE REVIEW

File: modules/ROOT/pages/design-import-files.adoc [hunk 1]
```diff
@@ -41,8 +41,8 @@ The fragments that you import can be written in RAML 0.8 or RAML 1.0. You can im
 * To import an API specification or API fragment from Exchange, together with its dependencies:
 . Open the *Import Asset from Exchange* dialog by using either of these methods:
 +
-** In the code editor only: If your project contains only the default file that API Designer created together with the project, click the linked text that appears in the *Files* pane.
-** In the code editor only: If your project contains at least one file that you have created, click the three-dot icon at the top of the left pane and select *Import from Exchange*.
+** *In the code editor only:* If your project contains only the default file that API Designer created together with the project, click the linked text that appears in the *Files* pane.
```
Comment (kbostrom) [inline modules/ROOT/pages/design-import-files.adoc]:
```suggestion
** *In the code editor only* 
If your project contains only the default file that API Designer created together with the project, click the linked text that appears in the *Files* pane.
```

File: modules/ROOT/pages/design-import-files.adoc [hunk 2]
```diff
@@ -16,8 +16,8 @@ The fragments that you import can be written in RAML 0.8 or RAML 1.0. You can im
 
 == Procedure
```
Comment (kbostrom) [inline modules/ROOT/pages/design-import-files.adoc]: This is quite a mixture of nested different formats. Consider reevaluating for making this easier on the audience's eyes....

File: modules/ROOT/pages/design-import-files.adoc [hunk 3]
```diff
@@ -41,8 +41,8 @@ The fragments that you import can be written in RAML 0.8 or RAML 1.0. You can im
 * To import an API specification or API fragment from Exchange, together with its dependencies:
 . Open the *Import Asset from Exchange* dialog by using either of these methods:
 +
-** In the code editor only: If your project contains only the default file that API Designer created together with the project, click the linked text that appears in the *Files* pane.
-** In the code editor only: If your project contains at least one file that you have created, click the three-dot icon at the top of the left pane and select *Import from Exchange*.
+** *In the code editor only:* If your project contains only the default file that API Designer created together with the project, click the linked text that appears in the *Files* pane.
+** *In the code editor only:* Click *Settings* in the top-right corner of the code editor and select *Import from Exchange*.
 ** In the visual editor only: If your project does not yet contain any resources or data types, click the linked text that appears in the *New API* pane.
```
Comment (kbostrom) [inline modules/ROOT/pages/design-import-files.adoc]:
```suggestion
** In the visual editor only
If your project does not yet contain any resources or data types, click the linked text that appears in the *New API* pane.
```
Note that this is not boldfaced like the previous bullet item is: standardize.

File: modules/ROOT/pages/design-import-files.adoc [hunk 4]
```diff
@@ -16,8 +16,8 @@ The fragments that you import can be written in RAML 0.8 or RAML 1.0. You can im
 
 == Procedure
 
-* In the code editor only: To import a file or the content of a .zip file that is stored on your computer:
-. In the left pane of the editor, click the dots to the right of *Files* and select *Import*.
+* *In the code editor only:* To import a file or the content of a .zip file that is stored on your computer:
```
Comment (kbostrom) [inline modules/ROOT/pages/design-import-files.adoc]:
```suggestion
* *In the code editor only* 
To import a file or the content of a .zip file that is stored on your computer:
```

File: modules/ROOT/pages/design-import-files.adoc [hunk 5]
```diff
@@ -41,8 +41,8 @@ The fragments that you import can be written in RAML 0.8 or RAML 1.0. You can im
 * To import an API specification or API fragment from Exchange, together with its dependencies:
 . Open the *Import Asset from Exchange* dialog by using either of these methods:
 +
-** In the code editor only: If your project contains only the default file that API Designer created together with the project, click the linked text that appears in the *Files* pane.
-** In the code editor only: If your project contains at least one file that you have created, click the three-dot icon at the top of the left pane and select *Import from Exchange*.
+** *In the code editor only:* If your project contains only the default file that API Designer created together with the project, click the linked text that appears in the *Files* pane.
+** *In the code editor only:* Click *Settings* in the top-right corner of the code editor and select *Import from Exchange*.
```
Comment (kbostrom) [inline modules/ROOT/pages/design-import-files.adoc]:
```suggestion
** *In the code editor only* 
Click *Settings* in the top-right corner of the code editor and select *Import from Exchange*.
```

File: modules/ROOT/pages/design-import-files.adoc (hunk 1 above)
Comment (rddheath) [reply modules/ROOT/pages/design-import-files.adoc]: Resolved in a different way.

File: modules/ROOT/pages/design-import-files.adoc (hunk 3 above)
Comment (rddheath) [reply modules/ROOT/pages/design-import-files.adoc]: Resolved in a different way.

File: modules/ROOT/pages/design-import-files.adoc (hunk 4 above)
Comment (rddheath) [reply modules/ROOT/pages/design-import-files.adoc]: Resolved in a different way.

File: modules/ROOT/pages/design-import-files.adoc (hunk 5 above)
Comment (rddheath) [reply modules/ROOT/pages/design-import-files.adoc]: Resolved in a different way.

**Pull request merged**